index.html

```diff
@@ -2,17 +2,188 @@
+                    </div>
+
+                    <i class="fa fa-bars text-light d-lg-none"></i>
+                </div>
+            </div>
+        </div>
+
+        <nav class="d-none d-md-none d-lg-block">
+            <div class="nav-links">
+                <ul class="d-flex main-nav align-items-center list-unstyled link-list font-roboto social border-bottom">
+                    <li class="border-end p-3">
+                        <a href="#">U.S.</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">World</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Business</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Tech & Science</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Culture</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Newsgeek</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Sports</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Health</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">The Debate</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Experts</a>
+                    </li>
+                    <li class="border-end">
+                        <a href="#">Vantage</a>
+                    </li>
+                    <li>
+                        <div>
+                            <form action="./">
+                                <input id="search" type="text" class="search font-roboto" placeholder="Search">
+                                <label for="search"><i class="fa fa-search"></i></label>
+                            </form>
+                        </div>
+                    </li>
+                </ul>
+
+                <ul
+                    class="sub-nav social header-title social font-roboto list-group list-group-horizontal list-unstyled border-bottom">
+                    <li><a href="#" class="black-header-title">TRENDING</a></li>
+                    <li><a href="#" class="header-title">JOE BIDEN</a></li>
+                    <li><a href="#" class="header-title">DONALD TRUMP</a></li>
+                    <li><a href="#" class="header-title">INAUGURATION</a></li>
+                    <li><a href="#" class="header-title">CORONAVIRUS</a></li>
+                    <li><a href="#" class="header-title">STIMULUS PACKAGE</a></li>
+                    <li><a href="#" class="header-title">VACCINE</a></li>
+                    <li><a href="#" class="header-title">CAPITOL RIOTS</a></li>
+                </ul>
+
+                <div class="y">
```

Comment:
[optional] For CSS best practices, you should use descriptive class names, `class="y"` is not descriptive at all, I would advise you to replace `y` with a descriptive name